src/pages/SettingsPage.js

```diff
@@ -38,24 +41,32 @@ const propTypes = {
 
     // Information about the current session (authToken, accountID, email, loading, error)
     session: PropTypes.shape,
+
+    // Information about the current session (authToken, accountID, email, loading, error)
+    currentURL: PropTypes.string,
 };
 
 const defaultProps = {
     myPersonalDetails: {},
     network: null,
     currentlyViewedReportID: '',
+    currentURL: '',
     session: {},
 };
 const SettingsPage = ({
-    myPersonalDetails, network, session, currentlyViewedReportID,
+    myPersonalDetails, network, session, currentlyViewedReportID, currentURL,
 }) => {
     // On the very first sign in or after clearing storage these
     // details will not be present on the first render so we'll just
     // return nothing for now.
     if (!myPersonalDetails || _.isEmpty(myPersonalDetails)) {
         return null;
     }
-
+    useEffect(() => {
+        if (currentURL === '/settings') {
+            hideSidebar();
+        }
+    }, [currentURL]);
```

Comment:
Sorry this isn't going to work please see my previous suggestion. Best would be to toggle the sidebar from the `HomePage` when the avatar is tapped. This has an issue on web. Also we are not using the `useEffect()` hook anywhere in the code so it would be preferable to avoid. Thanks.

https://user-images.githubusercontent.com/32969087/105085422-cb945b80-5a3b-11eb-9e86-d024998b0306.mp4